datafusion/proto/proto/datafusion.proto

```diff
@@ -508,6 +508,9 @@ message ScalarUDFExprNode {
 enum BuiltInWindowFunction {
   UNSPECIFIED = 0; // https://protobuf.dev/programming-guides/dos-donts/#unspecified-enum
   // ROW_NUMBER = 0;
+  // TODO
+  // if i will remove this
+  // should i renumber all variables?
   RANK = 1;
```

Comment:
@jcsherin  how to handle this ?
I want to remove rank, percent_rank and dense_rank from here ?